data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/JsonTypeDefinitionBeaconRestApi.java

```diff
@@ -322,7 +323,8 @@ private static RestApi create(
             .endpoint(new GetEth1VotingSummary(dataProvider, eth1DataProvider))
             .endpoint(new GetGlobalValidatorInclusion(dataProvider))
             .endpoint(new GetFinalizedStateSlotBefore(dataProvider))
-            .endpoint(new GetValidatorInclusion(dataProvider));
+            .endpoint(new GetValidatorInclusion(dataProvider))
```

Comment:
oh this is interesting, i put in a ticket on validator_inclusion, i'll have to revisit